components/eam/src/dynamics/se/stepon.F90

```diff
@@ -444,6 +446,12 @@ subroutine stepon_run2(phys_state, phys_tend, dyn_in, dyn_out )
          call outfld('DYN_MU',tmp_dyn_i(:,:,:),npsq,ie)
       enddo
    endif
+   if (hist_fld_active('DYN_PV')) then
```

Comment:
Based on discussion at tracers meeting, I think we should probably remove or comment this out for now, with a note indicating that this code is not working as expected.

Reply:
Output is now disabled